ui/src/Groups/Counter/Counter.view.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Body, Subheading } from '../../Elements/Text/index.js';
+
+const Counter = (props) => {
+  const { title, subtitle, count, countLimit, className, ...others } = props;
+  const classesToApply = `Counter${className ? ` ${className}` : ''}`;
+
+  let countValue = count;
+  if (count < 0) {
+    countValue = '00';
+  } else if (count < 10) {
+    countValue = '0' + count;
+  } else if (count > countLimit) {
+    countValue = countLimit + '+';
+  }
+
+  return (
+    <div {...others} className={classesToApply}>
+      <div className={'Counter__title'}>
+        <Subheading>{title}</Subheading>
+      </div>
+      <div className={'Counter__subtitle'}>
+        <Body>{subtitle}</Body>
+      </div>
+      <div className={'Counter__count'}>
+        <Body className={'Counter__countValue'}>{countValue}</Body>
```

Comment:
CSS override is making it behave like a `<Heading>` - I think it's safe to switch directly to it instead of `<Body>`

With provisor @matthew-chirgwin that the `<Heading>` will remain purely styling and not use `<h1>` etc.

Reply:
Was intended to be purley stylistic - with that style changing once we get time with design next week

Reply:
Given that I immediately override the `<Body>` styling with my own, and it's not clear if the styling for the Counter value should actually match headings, perhaps I shouldn't use either `<Heading>` or `<Body>`. They imply specific parts of the page, whereas I think the counter value is perhaps a different thing to be separately styled. Maybe `<Text>` needs a `CUSTOM` type modifier, or something like that..

Reply:
True, if this isn't considered to need "common" styling then I'd leave as plain content inside the parent `<div>` applying styling to parent.

Reply:
Good plan. Done.